module/applications/actor/npc-sheet-2.mjs

```diff
@@ -190,6 +193,41 @@ export default class ActorSheet5eNPC2 extends ActorSheetV2Mixin(ActorSheet5eNPC)
     context.hasClasses = classes.length;
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Prepare spellcasting data for display.
+   * @param {object} context  The display context.
+   * @protected
+   */
+  _prepareSpellcasting(context) {
+    const { abilities, attributes, bonuses, details } = this.actor.system;
+    context.spellcasting = [];
+    const msak = simplifyBonus(bonuses.msak.attack, context.rollData);
+    const rsak = simplifyBonus(bonuses.rsak.attack, context.rollData);
+    // TODO: Consider if we want to handle multiclassing for NPC spellcasters.
```

Comment:
Though we don't have anything to prevent it now, it would probably just be best to not allow multiclassing on NPCs.